src/Umbraco.Web.UI/Umbraco/Views/AuthorizeUpgrade.cshtml

```diff
@@ -1,16 +1,17 @@
 ﻿@using Umbraco.Core
-@using ClientDependency.Core
-@using ClientDependency.Core.Mvc
 @using Umbraco.Web.Composing
 @using Umbraco.Web
+
 @inherits System.Web.Mvc.WebViewPage<Umbraco.Web.Editors.BackOfficeModel>
 @{
     Layout = null;
 
-    Html
-        .RequiresCss("assets/css/umbraco.css", "Umbraco")
-        .RequiresCss("lib/bootstrap-social/bootstrap-social.css", "Umbraco")
-        .RequiresCss("lib/font-awesome/css/font-awesome.min.css", "Umbraco");
+    var runtimeMinifier = Current.RuntimeMinifier;
+
+    runtimeMinifier.RequiresCss("umbraco", 
```

Comment:
This needs to be a pre-defined bundle... as in, defined in c# on startup. View based declarations will not work in load balancing (see prev comments). I'll update.